LayoutTests/fast/mediastream/image-capture-get-photo-capabilities.html

```diff
@@ -20,9 +20,21 @@
             assert_equals(track.readyState, 'ended');
 
             const imageCapture = new ImageCapture(track);
-            return promise_rejects_dom(test, 'InvalidStateError', imageCapture.getPhotoCapabilities())
+            const promise = imageCapture.getPhotoCapabilities();
 
-        }, `getPhotoCapabilities() on an 'ended' track should throw "InvalidStateError"`);
+            let result;
+            promise.then(
+                (value) => { result = value; },
+                (error) => { result = error; }
+            );
+
+            Promise.resolve().then(() => {
+                assert_equals(result['name'], 'InvalidStateError');
+            });
+
+            await Promise.resolve();
```

Comment:
It would read better to write it as:
```
await Promise.resolve();
assert_equals(result['name'], 'InvalidStateError');
return promise;
```
Ditto for the other tests.